dist/src/assembly/bin/zeneventserver-functions.sh

```diff
@@ -64,6 +68,32 @@ wait_for_startup() {
     return 0
 }
 
+update_schema() {
+    zengc=$ZENHOME/bin/zenglobalconf
+    local dbname=$(${zengc} -p zep-db)
+    local dbtype=$(${zengc} -p zep-db-type)
+    local host=$(${zengc} -p zep-host)
+    local port=$(${zengc} -p zep-port)
+    local user=$(${zengc} -p zep-user)
+    local userpass=$(${zengc} -p zep-password)
+    /opt/zenoss/bin/zeneventserver-create-db --update_schema_only \
+	--dbhost $host --dbport $port \
+        --dbname $dbname --dbtype $dbtype \
+        --dbuser $user --dbpass "$userpass" || return "$?"
+    return 0
+}
+
+wait_and_update_schema() {
+    local port=$1
+    wait_for_startup $port || return $?
```

Comment:
Similar to above. Is there a reason to wait for ZEP to be running before trying to upgrade the DB?